guides/managing-current-user.md

```diff
@@ -64,14 +64,57 @@ export default Ember.Service.extend({
 
   load() {
     if (this.get('session.isAuthenticated')) {
-      return this.get('store').find('user', 'me').then((user) => {
+      return this.get('store').findRecord('user', 'me').then((user) => {
```

Comment:
We should probably have `queryRecord` and make what is described in _"Using a dedicated endpoint and returning actual user id in response"_ below the default recommendation as `findRecord` will lead to problems and thus shouldn't be in the guides at all.